test/unit/data/data.ts

```diff
@@ -83,7 +83,7 @@ export const requestWithPayloadCredentialAndScope: Request = {
 // @ts-ignore
```

Comment:
do we need this @ts-ignore ?

Reply:
@eoln it is required because the Request interface has many more fields. So in order to create an object with only the relevant fields, they used ts-ignore I guess.